samples/ChangeFeedManagement/Program.py

```diff
@@ -137,4 +137,4 @@ def run_sample():
         run_sample()
 
     except Exception as e:
-            print("Top level Error: args:{0}, message:N/A".format(e.args))
+        print("Top level Error: args:{0}, message:N/A".format(e.args))
```

Comment:
Python 2 requires only consistent indentation within a specific control block (so this is legal).  Python 3 requires consistency (size and tabs vs spaces) across a whole file.